synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -228,8 +229,10 @@ MainWindow::init_menus()
 
 void MainWindow::register_custom_widget_types()
 {
+	Widget_GammaPattern::register_type();
```

Comment:
Maybe we should start to use some ordering logic here: alphabetically or by "age" (creation?).

What do you think?

Reply:
I prefer alphabetically :)

Reply:
Making the changes